### PR TITLE
Advanced queries using public instances

### PR DESCRIPTION
**Pull request by armaneous**

### CODE REVIEW

**Comment by scvenema** [comment]
> The `world-open` queries are probably failing because of the `#{2**32 - 1}` ERB templating that just does math (2^32 - 1) to signify IPv4 fully open CIDR range (equivalent to `0.0.0.0/0`).

My understanding is that our API call to Rosetta to Rule Evaluator includes the ERB templating. So if there is a problem here, there should be a problem in using it as a stored query.

File: stack-audit-cloud-threats/services/config.rb [hunk 1]
```diff
@@ -184,3 +184,1092 @@
...
+                            'subnet' => [],
+                            'instance' => ['public_ip_address']
+                          })
+end
+
+coreo_aws_rule "public-instance-world-open-TCP-22" do
+  action :define
+  service :ec2
+  link "https://kb.securestate.vmware.com/aws-ec2-tcpportopen.html"
+  display_name "Public Instance TCP port is open - 22"
```
Comment (coolguru) [inline stack-audit-cloud-threats/services/config.rb]: can you add more meta data here? port 22 could be `SSH port 22. `

Comment (coolguru) [reply stack-audit-cloud-threats/services/config.rb]: same for others as well

**Comment by scvenema** [comment]
Passes manual rule syntax checks :-)

**Pull request merged**